boilerplate/_lib/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+ENV GOCILINT_VERSION="1.31.0"
+ENV GOCILINT_SHA256SUM="9a5d47b51442d68b718af4c7350f4406cdc087e2236a5b9ae52f37aebede6cb3"
+
+ENV OPERATOR_SDK_VERSION="0.16.0"
+ENV OPERATOR_SDK_SHA256SUM="3df782f341749f7962ab0fcfedd2961c18b21ad34ff7acd194b49a152f59abcb"
+
+ENV OPM_VERSION="1.13.8"
+ENV OPM_SHASUM="a48aa9d69b0be3439220e818edde4b36b3b9eceb2a058d86b4fdf0ca9dcd21c8"
+
+RUN curl -L -o golangci-lint.tar.gz https://github.com/golangci/golangci-lint/releases/download/v${GOCILINT_VERSION}/golangci-lint-${GOCILINT_VERSION}-linux-amd64.tar.gz && \
+    echo ${GOCILINT_SHA256SUM} golangci-lint.tar.gz && \
+    tar xzf golangci-lint.tar.gz golangci-lint-${GOCILINT_VERSION}-linux-amd64/golangci-lint && \
+    mv golangci-lint-${GOCILINT_VERSION}-linux-amd64/golangci-lint /usr/local/bin && rm -rf golangci-lint-${GOCILINT_VERSION}-linux-amd64 && \
+    rm -f golangci-lint.tar.gz && \
+    curl -L -o operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu && \
+    echo ${OPERATOR_SDK_SHA256SUM} operator-sdk | sha256sum -c && \
+    mv operator-sdk /usr/local/bin && \
+    curl -L -o opm https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm && \
+    echo ${OPM_SHASUM} opm | sha256sum -c && \
+    mv opm /usr/local/bin
```

Comment:
Can we move these one-massive-line commands into a script in the repo next to this Dockerfile? Or is that a chicken/egg problem?

Reply:
It can be done.